test/Sentry.Tests/SentryClientTests.cs

```diff
@@ -289,6 +290,62 @@ public void CaptureEvent_DisposedClient_ThrowsObjectDisposedException()
             _ = Assert.Throws<ObjectDisposedException>(() => sut.CaptureEvent(null));
         }
 
+        [Fact]
+        public void CaptureUserFeedback_Ignored_If_EventIdEmpty()
+        {
+            //Arrange
+            var sut = _fixture.GetSut();
+            var captureCalledEvent = new ManualResetEvent(false);
+            sut.Worker.When(x => x.EnqueueEnvelope(Arg.Any<Envelope>()))
+                .Do(_ => captureCalledEvent.Set());
+
+            //Act
+            sut.CaptureUserFeedback(new SentryUserFeedback(SentryId.Empty, "email", "comment"));
+
+            //Assert
+            Assert.False(captureCalledEvent.WaitOne(1));
+        }
+
+        [Fact]
+        public void CaptureUserFeedback_Ignored_If_Email_Is_NullOrEmpty()
+        {
+            //Arrange
+            var sut = _fixture.GetSut();
+            var captureCalledEvent = new ManualResetEvent(false);
+            sut.Worker.When(x => x.EnqueueEnvelope(Arg.Any<Envelope>()))
```

Comment:
Not sure why we need `ManualResetEvent` here given there's no threading going on.

The `Worker` is a mock. So you can just assert it was called.

Also, you're asserting it was **not** called?
`            Assert.False(captureCalledEvent.WaitOne(1));`

Reply:
Replaced by sut.Worker.DidNotReceive().EnqueueEnvelope(Arg.Any<Envelope>());